packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -402,7 +406,8 @@ export class ARRenderer extends EventDispatcher {
 
       if (this.threeRenderer.xr.getSession() != null) {
         this.projectionMatrix.copy(
-            this.threeRenderer.xr.getCamera(camera).projectionMatrix);
+            //@ts-ignore
+            this.threeRenderer.xr.getCamera().projectionMatrix);
```

Comment:
@Mugen87 I'm also not thrilled about this logic. It would be nice if `updateCamera()` also updated my camera's projection matrix and its inverse, though we probably don't want to invert on every frame. Or should I just forget about my camera and use `getCamera()` instead? This two cameras system where they are not quite in sync has not felt super ergonomic.

Reply:
Couldn't you do `const xrCamera = this.threeRenderer.xr.getCamera();` at the top of the file?

Reply:
I use my own camera internally for things like hotspot rendering, but yeah, I could probably make a `getCamera()` function that'll return the XR one when I'm in AR. That does sound less error-prone.